core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1022,45 +1022,63 @@
       "Amount of time for a single replication RPC call to last before failing"
           + " the attempt. See replication.work.attempts."),
   // Compactor properties
+  @Experimental
   COMPACTOR_PREFIX("compactor.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo compactor server."),
+  @Experimental
   COMPACTOR_PORTSEARCH("compactor.port.search", "true", PropertyType.BOOLEAN,
       "If the compactor.port.client is in use, search higher ports until one is available"),
+  @Experimental
   COMPACTOR_CLIENTPORT("compactor.port.client", "9101", PropertyType.PORT,
       "The port used for handling client connections on the compactor servers"),
+  @Experimental
   COMPACTOR_MINTHREADS("compactor.server.threads.minimum", "1", PropertyType.COUNT,
       "The minimum number of threads to use to handle incoming requests."),
+  @Experimental
   COMPACTOR_MINTHREADS_TIMEOUT("compactor.server.threads.timeout", "0s", PropertyType.TIMEDURATION,
       "The time after which incoming request threads terminate with no work available.  Zero (0) will keep the threads alive indefinitely."),
+  @Experimental
   COMPACTOR_THREADCHECK("compactor.server.threadcheck.time", "1s", PropertyType.TIMEDURATION,
       "The time between adjustments of the server thread pool."),
+  @Experimental
   COMPACTOR_MAX_MESSAGE_SIZE("compactor.server.message.size.max", "10M", PropertyType.BYTES,
       "The maximum size of a message that can be sent to a tablet server."),
   // CompactionCoordinator properties
+  @Experimental
   COORDINATOR_PREFIX("coordinator.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo compaction coordinator server."),
+  @Experimental
   COORDINATOR_THRIFTCLIENT_PORTSEARCH("coordinator.thrift.port.search", "false",
```

Comment:
This directly exposes "thrift" as the RPC implementation. It should probably not reference thrift.

Reply:
Removed 'thrift' from property names in 42d0c8445d964f079f39ef4c50d05559191982a3